Decorator_practice/src/LightModeName.java

```diff
@@ -0,0 +1,11 @@
+public class LightModeName extends BaseDecorator{
+
+    public LightModeName(IShowUserName wrappee) {
+        super(wrappee);
+    }
+    public String getUserName(String name) {
+        super.getUserName(name);
+        return "***" + name + "***" + "┬─┬ノ( º _ ºノ)";
```

Comment:
I was expecting each one of this ones AsteriskDecorator, italicDecorator and FlipperDecorator